demos/browser/README.md

```diff
@@ -36,7 +36,11 @@ the same AWS region as the meeting and have the following bucket policy:
     ]
 }
 ```
-
+In order to use the live connector function in the browser demo, you should provide the ivs channel endpoint and set environment variable 
```

Comment:
As commented earlier would be provide more helpful links here:
https://github.com/aws/amazon-chime-sdk-js/pull/2428#discussion_r968645678

Suggest re-phrasing it as mentioned below.

nit: ivs -> Amazon Interactive Video Service (IVS).

"To use the Amazon Chime SDK media pipelines live connector feature, provide an Amazon Interactive Video Service (IVS) channel endpoint and set the environment variable when running locally:
> In serverless demo deployment, the IVS channel is created.
```
export IVS_ENDPOINT=<rtmps://ingest-server-url:443/app/stream-key>
```
Check [Amazon Chime SDK launches live connector for streaming](https://aws.amazon.com/blogs/business-productivity/amazon-chime-sdk-launches-live-connector-for-streaming/) blog post for more information.

Correct me if wrong somewhere.

Reply:
Updated